libexec/rbenv-whence

```diff
@@ -24,6 +24,19 @@ whence() {
     path="$(rbenv-prefix "$version")/bin/${command}"
     if [ -x "$path" ]; then
       [ "$print_paths" ] && echo "$path" || echo "$version"
+    else
+      gem="$(rbenv-prefix "$version")/bin/gem"
+      if [ -x "$gem" ]; then
+        OLDIFS="$IFS"
+        IFS=':' gempaths=(`$gem environment gempath`)
```

Comment:
I think '$gem' here should be quoted.

There are a few more occurances of this.

Reply:
All occurrences of this are now replaced.